projects/jupyter.py

```diff
@@ -170,3 +171,44 @@ def read_parameters_from_source(source):
                 pass
 
     return parameters
+
+
+def get_notebook_output(experiment_id: str, operator_id: str):
+    """Get Experiment notebook output.
+
+    Args:
+        experiment_id (str): experiment id
+        operator_id (str): operator id
+
+    Returns:
+        dict: a dictonary with output.
+    """
+    operator_endpoint = f"experiments/{experiment_id}/operators/{operator_id}/Experiment.ipynb"
+
+    try:
+        r = SESSION.get(url=f"{URL_CONTENTS}/{operator_endpoint}").content
+        notebook_content = loads(r.decode("utf-8"))["content"]
+    except HTTPError as e:
+        status_code = e.response.status_code
+        if status_code == 404:
+            raise FileNotFoundError("The specified notebook does not exist")
+
+    for cell in notebook_content["cells"]:
+        try:
+            metadata = cell["metadata"]["papermill"]
+
+            if metadata["exception"] and metadata["status"] == "failed":
+                output = cell["outputs"][0]
+                traceback = remove_ansi_escapes(output["traceback"])
+
+                return {
+                    "cellType": cell["cell_type"],
+                    "executionCount": cell["execution_count"],
+                    "output": {
+                        "errorName": output["ename"],
+                        "errorValue": output["evalue"],
+                        "traceback": traceback,
+                    }
+                }
+        except KeyError:
+            pass
```

Comment:
@samborba se a execução for um sucesso, vai retornar None/null, né?

Não parece bom... Talvez você possa retornar um campo indicando status de sucesso/falha.

Outra possibilidade é transformar a API em "error_logs", e no caso de sucesso retornar 404.